### PR TITLE
Create membership.md

### PR DESCRIPTION
**Pull request by yaron2**
This PR adds the missing template to become an org member.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/membership.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### GitHub Username
+
+e.g. (at)example_user
+
+### Requirements
+
+- [ ] I have reviewed the community membership guidelines (https://github.com/dapr/community/blob/master/community-membership.md)
+- [ ] I have enabled 2FA on my GitHub account. See https://github.com/settings/security
+- [ ] I have subscribed to the [Gitter channel](https://gitter.im/dapr/community)
```
Comment (mchmarny) [inline .github/ISSUE_TEMPLATE/membership.md:18]: Dapr Gitter channel

File: .github/ISSUE_TEMPLATE/membership.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- [ ] I am actively contributing (any of the following apply: issues, discussions, PRs, reviews) to 1 or more Dapr repositories
```
Comment (mchmarny) [inline .github/ISSUE_TEMPLATE/membership.md:19]: Remove "actively"

**Review by mchmarny — CHANGES_REQUESTED** [review]
couple of comments

File: .github/ISSUE_TEMPLATE/membership.md (hunk 1 above)
Comment (yaron2) [reply .github/ISSUE_TEMPLATE/membership.md]: Done

File: .github/ISSUE_TEMPLATE/membership.md (hunk 2 above)
Comment (yaron2) [reply .github/ISSUE_TEMPLATE/membership.md]: Done

**Review by mchmarny — APPROVED** [review]
/lgtm

**Pull request merged**